### DISCUSSION TITLE
posts/2017-03-05-interviews/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#11] [Announcements]
# posts/2017-03-05-interviews/

The tech interview process is very, very broken.

https://rachel.fast.ai/posts/2017-03-05-interviews/

**Comment by warrengmartin** [comment]
Good read. 

Take home assignments resonates with me quite a bit. As a person who typically needs more time with tests, being able to complete a test within the comfort of a calm environment will showcase my abilities more accurately compared to the alternative. Someone breathing down my neck judging my every move, more often than not, will produce a bad performer.

I often wonder what the world would look like if apprenticeships were brought back into the mix. College graduates having the ability to in a field for a set number of hours, and hopefully compensation) to gain experience and breathe the same air as full-time time employees.

What are your thoughts on how hard the job market is on people with social anxiety and depression? These groups of people tend to have lower self-esteem, have trouble speaking with people in person, and have trouble connecting with people. All aspects an interviewer would see as "not qualified", but once hired, are great in the role.

Thanks.

**Reply by racheltho** [reply]
Thanks for reading!  You bring up two important issues. I think employers rewarding over-confidence and creating an artificial level of adversarial pressure are two ways that people with depression or anxiety are unfairly penalized in their job searches. It makes sense that a take-home would be a better alternative for this group.  I also realize that many people need extra time on tests, and that this is not a reflection of how they would do in the workplace.

I agree that it would be nice to see more apprenticeships.  In general, I think that most employers under-invest in training people who could make great employees if they were given some decent mentorship.
